ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -625,74 +670,53 @@ func init() {
 func main() {
 	flag.Parse()
 
+	// use in-cluster config if kubeConfig path is not passed as a flag
+	var client *kubernetes.Clientset
+	if KubeConfigPath == "" {
+		config, err := rest.InClusterConfig()
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	} else {
+		config, err := clientcmd.BuildConfigFromFlags("", KubeConfigPath)
+		if err != nil {
+			panic(err)
+		}
+		client = kubernetes.NewForConfigOrDie(config)
+	}
+
+	// instantiate PodInformer for Ray worker pods in the GKE cluster
+	tweakListOptionsFunc := func(options *metav1.ListOptions) {
+		options.LabelSelector = "ray.io/node-type=worker,app.kubernetes.io/created-by=kuberay-operator"
+	}
+	factory := informers.NewFilteredSharedInformerFactory(client, 5*time.Minute, metav1.NamespaceAll, tweakListOptionsFunc)
+	podLister := factory.Core().V1().Pods().Lister()
+
+	if podLister == nil {
+		klog.Error("Failed to initialize Pod Lister")
+		return
```

Comment:
This is in the main func, so returning will result in exit code 0. Use `os.Exit(1)` instead

Reply:
alternatively, you can use `klog.Fatal`

Reply:
Done in [7895a44](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/7895a442b08a6fd4d694240ad7a10c7aa89a8327)